templates/route.yaml

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements. See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership. The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: {{.Name}}
+  namespace: {{.Namespace}}
+  labels:
+    {{range $k,$v := .Labels }}
+    {{$k}}: {{$v}}
+    {{end}}
+spec:
+  to:
+    kind: Service
+    name: {{.SvcName}}
+    weight: 100
+  port:
+    {{range $k,$v := .Ports }}
+    targetPort: {{$k}}
```

Comment:
You are only using `key` here, what about the `value`? Might make sense to set the port name (`$k`) and port number (`$v`) - what do you think?

Reply:
Since the port name that gets set in the Service gets pulled here as key I kept it like this, Should I only pull `$k`?

Reply:
Nah, I think it is ok, let's keep at as is, since it works:)